TLM/TLM/UI/TrafficManagerTool.cs

```diff
@@ -459,9 +474,13 @@ void DefaultRenderOverlay(RenderManager.CameraInfo cameraInfo)
                     InfoManager.SubInfoMode.Default);
             }
             ToolCursor = null;
-            bool elementsHovered = DetermineHoveredElements();
+            bool elementsHovered = DetermineHoveredElements(activeLegacySubTool_ is not LaneConnectorTool);
```

Comment:
°O° c#9

Reply:
Yup a bit more readable than `!(activeLegacySubTool_ is LaneConnectorTool)`